client/components/login/login.test.js

```diff
@@ -92,20 +103,72 @@ describe("<Login /> interactions", () => {
           },
         });
       })
+      .mockImplementationOnce(() => {
+        return Promise.reject({
+          status: 500,
+          statusText: "Internal server error",
+          response: {
+            data: {
+              detail:	"Internal server error",
+            }
+          }
+        });
+      })
+      .mockImplementationOnce(() => {
+        return Promise.reject({
+          response: {
+            data: {}
+          },
+          status: 504,
+          statusText: "Gateway Timeout"
+        });
+      })
```

Comment:
- Added this lines to test `500 error` and `server unvailable`
- Added this data in `Promise.reject` arguments for `console.error` to show error correctly